adapters/Maio/MaioAdapter/GADMMaioConstants.h

```diff
@@ -10,7 +10,7 @@
 
 static NSString *const kGADMMaioErrorDomain = @"com.google.mediation.maio";
 
-static NSString *const kGADMMaioAdapterVersion = @"1.5.4.0";
+static NSString *const kGADMMaioAdapterVersion = @"2.0.0.0";
```

Comment:
🎨 I want to distinguish it from the canonical sdkv2 version by the build metadata.
e.g.
```suggestion
static NSString *const kGADMMaioAdapterVersion = @"2.0.0.0+openbidding";
```
This is because I want the version to be `2.0.0.0` when it is later replaced with the canonical sdkv2.